src/chromo/models/__init__.py

```diff
@@ -35,3 +36,89 @@
     "Pythia6",
     "Pythia8",
 )
+
+
+# This is a workaround of a problem with "fixed" wheels
+# which is produced by "delocate" package on cibuildwheels
+# It changes Python -> the path of libpython in the "*.so" files
+# using standard "otool" and "install_name_tool" routines
+def fix_macos_installation(logfile):
+    import subprocess
+    import re
+    from pathlib import Path
+    from distutils.sysconfig import get_config_var
+
+    models_dir = Path(__file__).parent
+    for ext_file in models_dir.glob("*.so"):
+        ext_file = str(ext_file)
+
+        # Get libraries required by extension module
+        cmd = ["otool", "-L", ext_file]
+        result = subprocess.run(cmd, stdout=subprocess.PIPE)
+        lib_deps = result.stdout.decode("utf-8")
+
+        # Write to the logfile the list of libraries
+        # required by extension module before fixing
+        message = f"\n----------\nFixing {ext_file}:\nBefore\n{lib_deps}"
+        with open(logfile, "a") as lf:
+            lf.write(message)
+
+        # Search for "Python" shared library
+        # among list of required libraries
+        search_res = re.search(
+            "^(.*python.*dylib|.*Python).*$", lib_deps, flags=re.MULTILINE
+        )
+
+        # Ignore if Python shared library is not found
+        if search_res is None:
+            message = (
+                'NOT FIXED: Paths  with "Python" or "python*.dylib" strings '
+                "isn't found inside shared library file"
+            )
+            with open(logfile, "a") as lf:
+                lf.write(message)
+            continue
+
+        # Path to Python shared libraries found in the extension file
+        python_lib_in_file = search_res.group(1).strip()
+        # Path to Python shared library found on the system
+        python_lib_real = str(next(Path(get_config_var("LIBDIR")).glob("libpython*")))
```

Comment:
I don't have it in pre-commit checks, because of 

```
[flake8]
max-line-length = 90
```

section in setup.cfg

In VS code I turned on this option in flake8 section by adding `--max-line-length=90` to args.